sys/src/9/amd64/mmu.c

```diff
@@ -649,7 +649,7 @@ vmap(uintptr_t pa, usize size)
 	DBG("vmap(%#p, %lu) pc=%#p\n", pa, size, getcallerpc());
 
 	if(machp()->machno != 0)
-		print("vmap: machp()->machno != 0");
+		print("vmap: machp()->machno != 0\n");
```

Comment:
This is generally done to avoid having stuff scroll off the screen. Consider a delimiter but if you newline that's fine too.